server/sonar-server/src/main/java/org/sonar/server/measure/ws/MeasureValueFormatter.java

```diff
@@ -39,8 +39,8 @@ static String formatMeasureValue(MeasureDto measure, MetricDto metric) {
       case BOOL:
         return formatBoolean(doubleValue);
       case INT:
-      case MILLISEC:
         return formatInteger(doubleValue);
```

Comment:
I'm wondering if formatInteger() is really useful ?
Couldn't we replace it by formatLong ?